src/crimson/os/seastore/transaction.h

```diff
@@ -165,7 +168,18 @@ class Transaction {
     retired_gate_token.reset(initiated_after);
     conflicted = false;
   }
-
+  bool in_write_set(paddr_t addr) {
```

Comment:
replace with std::any_of()?

Reply:
@ronen-fr sorry, i pushed a wrong branch. the other 2 commits are not related. just dropped them from this branch.